assets/models/system/phi-3.5-mini-128k-instruct/spec.yaml

```diff
@@ -31,6 +34,9 @@ tags:
   author: microsoft
   SharedComputeCapacityEnabled : ""
   hiddenlayerscanned : ""
+  maas-finetuning: true
```

Comment:
Can we please hold off on this! maas side of things need more work lets not add these tags yet. Other thing is we may only want to do first on MaaP

Reply:
is MaaS Sanity run not enough ? If not could we remove this particular tag and release the model for MaaP ?

Reply:
removing maas tags, keeping it to maap only